src/info.rs

```diff
@@ -0,0 +1,87 @@
+use crate::config::Config;
+use crate::fmt::{date, opt, print_indent};
+
+use failure::Error;
+use std::collections::HashMap;
+use term_size::dimensions_stdout;
+
+pub fn info(conf: &Config, pkgs: &[String], verbose: bool) -> Result<(), Error> {
+    let mut hm = HashMap::with_capacity(pkgs.len());
+
+    for chunk in pkgs.chunks(150) {
+        let pkgs = raur::info(&chunk)?;
+        for pkg in pkgs {
+            hm.insert(pkg.name.clone(), pkg);
+        }
+    }
+
+    let mut info = Vec::with_capacity(hm.len());
```

Comment:
hm?)
OK, jokes aside, maybe rename it to a longer name, like `details` or `name_to_pkg` or something?